### PR TITLE
docs: add a feature roadmap

### PR DESCRIPTION
**Pull request by nicks**
Hello @dbentley,

Please review the following commits I made in branch nicks/roadmap:

6aa77658d94a7057796c89d41b0d63be7b7717e7 (2018-12-07 13:21:26 -0500)
docs: add a feature roadmap

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Roadmap
+
+Welcome to the future!
+
+This page is for features that the Tilt team has concrete
```
Comment (dbentley) [inline ROADMAP.md]: "concrete plans" makes it sound like we have time frames; is there a way to say "we know about this and have some idea how"?

Comment (nicks) [reply ROADMAP.md]: "plaster plans"?

Comment (dbentley) [reply ROADMAP.md]: Hahahahaha.

"There's a lot we could do; we'd like feedback in how to prioritize these options based on what would be most useful"

Comment (dbentley) [reply ROADMAP.md]: (anyway, yeah, looks good)

**Pull request merged**